client/src/components/feddback-comment.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
```

Comment:
I think it's better to create "feedback" package in /components and name this file just "`comment.js`". It will not duplicate in other component names.

Reply:
You speak about create new folder feedback in components? And rename feedback-comment-->comment, feedback-send-->send?

Reply:
Yes, I do.

I would do so.